### PR TITLE
Csp 52 add standard regulated

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard
+{
+    public class ConfirmNewRegulatedStandardSubmitModel
+    {
+        public bool? IsCorrectStandard { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs:5]: IsApprovedByRegulator

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs:1]: For consistency, I would keep this class in the same file unless there is another deriving class.

**Review by DevenKShah — APPROVED** [review]
Looks good

File: src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.Roatp.CourseManagement.Web/Models/AddAStandard/ConfirmNewRegulatedStandardSubmitModel.cs]: Sounds good

**Review by DevenKShah — DISMISSED** [review]

**Review by DevenKShah — DISMISSED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**